kvserver: test to add store to running node

andrewbaptist opened the pull request:
Epic: none

Release note: None

Commit:
kvserver: test to add store to running node

Epic: none

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/137533)
<!-- Reviewable:end -->

Commit:
server: remove extra param

We were unnessarrily passing the stopper to initializeAdditionalStores.
This commit removes it.

Epic: CRDB-41111

Release note: None
